### ISSUE TITLE
[Feature idea] Add a keybind to refresh only the selected feed

### ISSUE THREAD

**Issue by Kabouik** [xgi/castero#133]
Refreshing all feeds can be long when many feeds have been added, and there is even a builtin option to ask for confirmation when many feeds will be updated. It would make sense to allow refreshing just the current feed when one knows in advance what one wants to listen, but doesn't necessarily want to update everything. 

Would it be possible to add a new keybinding and change the current as follows?
```
r         - reload/refresh selected feed
R         - reload/refresh all feeds
```
I use `castero` on a phone where the connection is sometimes weak, meaning refreshing everything is not only long, but is also more likely to fail if there are many things to refresh, so this would make sense for that use case too.

[Edit] Also, speaking of additional keybindings, I think a `X         - clear the download directory` option (with confirmation) would be pretty nice too, so that users can delete their downloads without leaving the application. ~~Could be extended as described above with a `x         - delete the selected episode from the download directory`, by the way.~~ Just realized this is already implemented on `s`, but not detailed in the help.

**Comment by xgi** [comment]
Good suggestions. I've added a keybind to reload only the selected feed. I did swap the defaults from what you said, though: `r` still reloads all feeds, `R` reloads the selected one. These can be modified in the config.

I also reworked the save/delete keybinds, since it was unintuitive. `s` is now only for saving (downloading) an episode or feed. `x` was added to delete either a single saved episode or all saved episodes in the selected feed. Confirmation prompts appear in certain cases.

Changes made with #134. It's been a while since I had a release, so I'll probably tick the version number this weekend.

Thanks!
